src/Catalog/Persistence/StringStorageContentWithAccessCondition.cs

```diff
@@ -0,0 +1,22 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using Microsoft.WindowsAzure.Storage;
+
+namespace NuGet.Services.Metadata.Catalog.Persistence
+{
+    public class StringStorageContentWithAccessCondition : StringStorageContent
```

Comment:
>StringStorageContentWithAccessCondition [](start = 17, length = 39)

This is sort of "request" content. XML docs plz

Reply:
Added docs to `StringStorageContentWithAccessCondition`, `StringStorageContentWithETag`, and `DeleteRequestOptions`.